docs/README.md

```diff
@@ -34,8 +34,8 @@ Turn any Google Sheet into a database.
 [Mental Models](https://mentalmodels.co)<br />
 Mental models are unchanging fundamentals for a particular subject, popularised by Charlie Munger.
 
-[Quick, Font!](https://mentalmodels.co)<br />
+[Quick, Font!](https://quickfont.xyz)<br />
 Some good fonts for prototyping or snazzing up an prototype (without having to go through the pain of downloading & installing).
 
 [Buy Meth](https://sheetmetal.io)<br />
```

Comment:
Beat me to it, saw the mismatched links as well. The link for Buy Meth is also incorrect, correction below:

```suggestion
[Buy Meth](https://buymeth.com/)<br />
```